.eslintrc.js

```diff
@@ -10,5 +10,9 @@ module.exports = {
 			'estree'
 		],
 		'svelte3/compiler': require('./compiler')
+	},
+	// workaround for https://github.com/typescript-eslint/typescript-eslint/issues/1824 and rely solely on 'indent' rule
+	rules: {
+		"@typescript-eslint/indent": "off"
```

Comment:
We should probably change this in the common config:

https://github.com/sveltejs/eslint-config/blob/e8a9b27cd3f7aa66388474412b1a5c11c5a44ade/index.js#L31

Reply:
sounds good. i've opened https://github.com/sveltejs/eslint-config/pull/1 and removed the custom rule i proposed initially here in 52eee5ed7.